### PR TITLE
Build webhook events properly

### PR DESCRIPTION
**Pull request by evelyn9191**
Solves https://github.com/rossumai/elisctl/issues/97

### CODE REVIEW

File: elisctl/webhook.py [hunk 1]
```diff
@@ -127,14 +128,13 @@ def change_command(
         if active is not None:
             data["active"] = active
         if events:
-            data["events"] = list(events)
+            data["events"] = [event for event in events]
```
Comment (proste) [inline elisctl/webhook.py:131]: i am wondering what is the difference, in which case this behaves differently than the original code (i am more curious than suspicious).

Comment (evelyn9191) [reply elisctl/webhook.py:131]: When `list(events)` is used, the response from Elis is:
`Error: Invalid response [https://api.elis.rossum.ai/v1/webhooks/123]: {"events": {"0": ["\"['annotation_status']\" is not a valid choice."]}}`
When `[event for event in events]` is used, the request is processed correctly as JSON, not with the escape backslash.

Comment (proste) [reply elisctl/webhook.py:131]: Maybe I am slower... sorry for that... but I am unable to reproduce it - i have read the original [issue](https://github.com/rossumai/elisctl/issues/97) which reports the following not to work
```
elisctl webhook change -e annotation status --config-url http://someurl.com
```
it seems to me the command should be
```
elisctl webhook change -e annotation_status --config-url http://someurl.com <some_id>
                                    ^
```
and then it works... lastly, when multiple events should be watched
```
elisctl webhook change -e annotation_status -e annotation_content --config-url http://someurl.com <some_id>
```
works for me as well (with expected results)

Comment (evelyn9191) [reply elisctl/webhook.py:131]: Actually, it was solved in https://github.com/rossumai/elisctl/pull/102
Closing this PR.

**Comment by evelyn9191** [comment]
Solved by https://github.com/rossumai/elisctl/pull/102

**Pull request closed without merging**